internal/http/client.go

```diff
@@ -202,6 +209,10 @@ func (c *NewRelicClient) setHeaders(req *retryablehttp.Request) {
 
```

Comment:
Should not send the APIKey header if explicitly said to use the Personal API Key

Reply:
👍